### ISSUE TITLE
ValueArray should have a .shape attribute

### ISSUE THREAD

**Issue by DanielSank** [labrad/pylabrad#110]
just like ndarray.

**Comment by chrismacklin** [comment]
+1

.dtype would be great as well.

An easy solution (though I don't know what the python pros think of this approach) is to define a ```__getattr__``` for ValueArray as
```python
def __getattr__(self, attr):
    return getattr(self._value, attr)
```

Then a ValueArray can be asked for any property of the underlying ndarray.  This might be confusing and have unintended side effects.

I could also just go ahead and implement shape and dtype properties for ValueArray and be done with it.

**Comment by DanielSank** [comment]
@chrismacklin go for it.

**Comment by chrismacklin** [comment]
Implemented in #125

**Comment by ejeffrey** [comment]
Closed via #125
